### PR TITLE
feat:add compose core

### PR DESCRIPTION
**Pull request by mtavano**

### CODE REVIEW

File: infra/dev/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  reporter-nodes:
+    image: darchlabs/reporter-dev
+    env_file:
+      - ./reporter.nodes.env
+    depends_on:
+      - nodes
+      - redis
+
+
+# data sotores
```
Comment (cagodoy) [inline infra/dev/docker-compose.yml]: fix typo

**Comment by cagodoy** [comment]
ok, merge

**Comment by nicobarra** [comment]
.env files should be only the variables name

**Comment by nicobarra** [comment]
Mint 🌱

**Pull request merged**